mllib/src/main/scala/org/apache/spark/mllib/evaluation/MulticlassMetrics.scala

```diff
@@ -139,6 +139,7 @@ class MulticlassMetrics @Since("1.1.0") (predictionAndLabels: RDD[(Double, Doubl
    * Returns precision
    */
   @Since("1.1.0")
+  @deprecated("Use accuracy.", "2.0.0")
   lazy val precision: Double = tpByClass.values.sum.toDouble / labelCount
```

Comment:
Sorry @wangmiao1981 can we do one more thing -- let's have all the deprecated methods call `accuracy`. Right now this (simple) expression is duplicated. And, note in precision's docs that it equals accuracy.